packages/react-dom/src/events/__tests__/SyntheticEvent-test.js

```diff
@@ -18,19 +17,23 @@ describe('SyntheticEvent', () => {
   var createEvent;
 
   beforeEach(() => {
-    // TODO: can we express this test with only public API?
-    SyntheticEvent = require('events/SyntheticEvent').default;
     React = require('react');
     ReactDOM = require('react-dom');
     ReactTestUtils = require('react-dom/test-utils');
 
-    createEvent = function(nativeEvent) {
-      var target = require('../getEventTarget').default(nativeEvent);
-      return SyntheticEvent.getPooled({}, '', nativeEvent, target);
+    createEvent = (nativeEvent, callback) => {
```

Comment:
Can we get rid of this helper and just inline calls in each test? I don't mind some repetition.